presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -44,7 +43,6 @@
 {
     public static final String PRODUCT_TEST_LAUNCHER_STARTED_LABEL_NAME = Environment.class.getName() + ".ptl-started";
     public static final String PRODUCT_TEST_LAUNCHER_STARTED_LABEL_VALUE = "true";
-    public static final String PRODUCT_TEST_LAUNCHER_NETWORK = "ptl-network";
```

Comment:
What happened to network name? We still want to use that name, don't we?

Reply:
> "principle of least astonishment"

This is very subjective rule.

> network per environment,

This is fine to me.

Reply:
> per environment

is this PR about reuse, or about allowing multiple environments?

Reply:
the second is consequence of first: to reuse, we should typically avoid killing containers, so some isolation is needed, hence prefixes in network/container names, and default `--bind=false` in reuse mode